### DISCUSSION TITLE
adguardhome无法重定向

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#86] [Bugs/问题反馈]
新编的固件，adguardhome显示运行中未重定向，设置和以前一样，重定向那里显示问号，不知是权限问题还是怎样

**Comment by Boos4721** [comment]
any log?

**Reply by ghost** [reply]
大佬我研究了一下，在adg手动设置里把端口改成自己设置的端口就可以正常重定向，按理说这应该是配置adg时自动修改的，不知道是adg最新内核的bug还是固件权限的问题，待后续观察

**Reply by Boos4721** [reply]
> 大佬我研究了一下，在adg手动设置里把端口改成自己设置的端口就可以正常重定向，按理说这应该是配置adg时自动修改的，不知道是adg最新内核的bug还是固件权限的问题，待后续观察

OK
